src/views/shared/page-header/page-header.view.jsx

```diff
@@ -31,7 +32,10 @@ function PageHeader ({
               <ArrowBackIcon />
             </button>
           )}
-          <h1 className={classes.title}>{title || ''}</h1>
+          <div className={classes.titleWrapper}>
+            <h1 className={classes.title}>{title || ''}</h1>
+            <h4 className={classes.subTitle}>{subTitle || ''}</h4>
```

Comment:
Instead of displaying its space if the `subtitle` is not there, I would better do:

```js
{subtitle && <h4 className={classes.subTitle}>{subTitle}</h4>}
```